config.sh

```diff
@@ -32,13 +32,11 @@ if [[ ! -d "${install_dir}/spack" ]]; then
     git clone git@github.com:spack/spack.git -b $version $install_dir/spack
 fi
 
-echo "Installing mch packages &" $hostname "config files"
-
-if [[ -n ${reposdir} ]] && [[ ! -f "${reposdir}/repos.yaml" ]]; then
-    echo " - $PWD" >> repos.yaml
-    cp repos.yaml $reposdir/
-fi
+echo "Installing mch packages on $install_dir/spack/etc/repos.yaml"
+echo " - $PWD" >> repos.yaml
+cp repos.yaml $install_dir/spack/etc/spack
```

Comment:
I probably didnt understand the purpose of the -r option. If I install the repos.yaml out of the spack/etc... 
my instance will not find the mch packages... So I was proposing here to harcode the installation of the repos.yaml inside the user spack installation. Is that a good idea?

Reply:
alternatively we can also put it under $user/.spack